pymc3/tuning/starting.py

```diff
@@ -171,3 +176,126 @@ def allinmodel(vars, model):
     notin = [v for v in vars if v not in model.vars]
     if notin:
         raise ValueError("Some variables not in the model: " + str(notin))
+
+
+
+class Monitor(object):
```

Comment:
any way to sanity check that this works?  i seems surprisingly non-brittle, but I'd still worry about it not following api changes if it isn't tested.